docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,313 @@
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+      .. empty-div:: arrow-down center-block invisible
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Live
+
+      .. noscript-content::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+      .. tabs::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
+
+
+      .. empty-div:: arrow-down arrow-offset center-block
+
+.. tutorial-step::
+
+   Step 2: Deploying a demo app
+   ============================
+   
+   .. tutorial-step-condensed::
+
+      You should have flocker-cli installed on your laptop and flocker-node installed on some servers: either VMs on your laptop, or real instances on cloud infrastructure.
+      Now you can try our simple tutorial: a Python web application and a Redis server.
+
+      .. code-block:: console
+
+         $ git clone https://github.com/clusterhq/flocker-quickstart
+         $ cd flocker-quickstart
+
+      fig.yml
+      -------
+
+      .. code-block:: yaml
+
+         web:
+           image: clusterhq/flask
+           links:
+            - "redis:redis"
+           ports:
+            - "80:80"
+         redis:
+           image: dockerfile/redis
+           ports:
+            - "6379:6379"
+           volumes: ["/data"]
+
+
+      deployment-node1.yml
+      --------------------
+
+      .. code-block:: yaml
+
+         "version": 1
+         "nodes":
+           "172.16.255.250": ["web", "redis"]
+           "172.16.255.251": []
+
+
+      The fig.yml describes your distributed application.
+      The deployment.yml describes which containers to deploy where.
+      If you are using real servers on AWS, you'll need to change the IP addresses in the deployment file.
+
+      .. code-block:: console
+
+         $ flocker-deploy deployment-node1.yml fig.yml
```

Comment:
`flocker-deploy` is not `$PATH` at this point.  This suggests we really actually do just want to activate the virtualenv.